src/envo/envo-edit.owl

```diff
@@ -11152,22 +11134,22 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_00000462> "polygenetic volcanic field"^^xsd:string)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000462> <http://purl.obolibrary.org/obo/ENVO_00000354>)
 
-# Class: <http://purl.obolibrary.org/obo/ENVO_00000463> (harbor)
+# Class: <http://purl.obolibrary.org/obo/ENVO_00000463> (harbour)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Harbor"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000463> "A place where ships may shelter from the weather or are stored."^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Harbor"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000463> "A landform which provides shelter from atmospheric weather and marine currents, waves, and other dynamics."^^xsd:string)
```

Comment:
perhaps 'a shoreline landform...'   otherwise any cave would meet this definition.